molecule/default/verify.yml

```diff
@@ -0,0 +1,11 @@
+---
+- name: Check whether rsyslog is configured to send data to the Alert Logic agent
+  hosts: all
+  become: yes
+
+  tasks:
+    - name: Check whether rsyslog is configured to send data to the Alert Logic agent
+      shell: grep -Fxq "*.* @@127.0.0.1:1514;RSYSLOG_FileFormat" /etc/rsyslog.d/alert-logic-agent.conf
+      changed_when: false
```

Comment:
2 items
- Is the changed_when necessary for tasks in the verify stage?
- lets also add a command task to check that the agent is running; `pgrep -f /var/alertlogic/lib/agent/bin/al-agent` will exit cleanly if the agent is running, and throw an error if not.

Reply:
>Is the changed_when necessary for tasks in the verify stage?

Nope.

>lets also add a command task to check that the agent is running
Done.